### PR TITLE
(SIMP-2555) Update nfs docs to use simp_nfs

### PR DESCRIPTION
**Pull request by jeefberkey**
SIMP-2555 #close

### CODE REVIEW

File: docs/user_guide/HOWTO/NFS.rst [hunk 1]
```diff
@@ -124,49 +137,48 @@ default.yaml
...
-  simp::nfs::export_home::create_home_dirs: true
 
   classes:
-    - 'simp::nfs::export_home'
-    - 'simp::nfs::home_client'
-
-Client
-^^^^^^
+    - simp_nfs
+    - simp_nfs::create_home_dirs
```
Comment (lnemsick-simp) [inline docs/user_guide/HOWTO/NFS.rst]: why is this a verb

File: docs/user_guide/HOWTO/NFS.rst [hunk 2]
```diff
@@ -198,22 +207,22 @@ Server
 .. code-block:: yaml
 
   nfs::is_server: true
-  simp::nfs::export_home::create_home_dirs: true
+  simp_nfs::create_home_dirs: true
```
Comment (lnemsick-simp) [inline docs/user_guide/HOWTO/NFS.rst]: This is a parameter and a class?  A bit confusing

File: docs/user_guide/HOWTO/NFS.rst [hunk 3]
```diff
@@ -77,16 +91,15 @@ In ``site/manifests/nfs_client.pp``:
     mount { "/mnt/nfs":
       ensure  => 'mounted',
       fstype  => 'nfs4',
-      device  => '<your_server_fqdn>:/srv/nfs_share',
-      options => 'sec=sys'
+      device  => 'puppet.simp.test:/var/nfs_share',
```
Comment (lnemsick-simp) [inline docs/user_guide/HOWTO/NFS.rst]: should this be your.server.fqdn instead of puppet.simp.test?

File: docs/user_guide/HOWTO/NFS.rst [hunk 4]
```diff
@@ -124,49 +137,48 @@ default.yaml
 
 .. code-block:: yaml
 
-  nfs::server:             "your.server.fqdn"
-  nfs::server::client_ips: "%{alias('client_nets')}"
-  nfs::simp_iptables:      true
-  nfs::simp_krb5:          false
+  nfs::is_server: false
+  nfs::client::stunnel::nfs_server: <your nfs server>
+  simp_nfs::home_dir_server: <your nfs server>
```
Comment (lnemsick-simp) [inline docs/user_guide/HOWTO/NFS.rst]: Inconsistent references to fqdn's.  Perhaps would be easiest if in the beginning, of the readme, we say that the code below assumes the NFS server is nfs.server.your.domain and NFS client is nfs.client.your domain.  Inconsistent naming is confusing to newbies like me.

File: docs/user_guide/HOWTO/NFS.rst [hunk 5]
```diff
@@ -29,24 +29,36 @@ In ``site/manifests/nfs_server.pp``:
...
 
-  class site::nfs_server {
+  class site::nfs_server (
+    Boolean          $kerberos     = simplib::lookup('simp_options::kerberos', { 'default_value' => false }),
+    Simplib::Netlist $trusted_nets = simplib::lookup('simp_options::trusted_nets', { 'default_value' => ['127.0.0.1'] }),
+  ){
+    include '::nfs'
+
+    if $kerberos {
+      $security = 'krb5p'
```
Comment (nick-markowski) [inline docs/user_guide/HOWTO/NFS.rst]: I don't see $security used anywhere.

**Pull request merged**